Source/WebKit/GPUProcess/ShapeDetection/RemoteFaceDetector.cpp

```diff
@@ -64,7 +64,7 @@ void RemoteFaceDetector::detect(WebCore::RenderingResourceIdentifier renderingRe
         return;
     }
 
-    m_backing->detect(*sourceImage, WTFMove(completionHandler));
+    Ref { m_backing }->detect(*sourceImage, WTFMove(completionHandler));
```

Comment:
m_backing is used pretty often and other similar classes implements `protectedBacking()`. I think we should have it in each classes.

Reply:
The pattern that we have adopted for now when writing these smart pointer adoption patches is to add `protected` functions when there is more than one call site where it will be used. Otherwise we just add `Ref { }` locally like in this case. It saves time when writing these adoption patches. 

Other files do implement `protectedBacking()` (I added many of these myself), but you'll notice that it's used in multiple places in those files. In this file, there's only one call site.